### PR TITLE
[SMALLFIX] Split MasterInfoIntegrationTest

### PR DESCRIPTION
**Pull request by uronce-cc**
Split `MasterInfoIntegrationTest` into `file/FileSystemMasterIntegrationTest` and `rawtable/RawTableMasterIntegrationTest` to make some `public` methods in `FileSystemMasterIntegrationTest` public in package level.

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/master/rawtable/RawTableMasterIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  @Ignore
+  @Test
+  public void createRawTableTest() throws InvalidPathException, FileAlreadyExistException,
+        TableColumnException, FileDoesNotExistException, TachyonException {
+    mRawTableMaster.createRawTable(new TachyonURI("/testTable"), 1, null);
+    Assert.assertTrue(
+        mFsMaster.getFileInfo(mFsMaster.getFileId(new TachyonURI("/testTable"))).isFolder);
+  }
+
+  @Test(expected = TableColumnException.class)
```
Comment (apc999) [inline integration-tests/src/test/java/tachyon/master/rawtable/RawTableMasterIntegrationTest.java]: This is the old way of junit to expect exception class.
You may leave a TODO to clean them up and use the new way using ` ExpectedException.none();`

File: servers/src/main/java/tachyon/master/file/FileSystemMaster.java [hunk 2]
```diff
@@ -429,10 +427,8 @@ public void completeFile(long fileId) throws FileDoesNotExistException, BlockInf
     }
   }
 
-  // TODO(cc) Make it protected instead of public for use in tests, after splitting up
-  // MasterInfoIntegrationTest
-  public void completeFileInternal(List<Long> blockIds, long fileId, long fileLength, long opTimeMs)
-      throws FileDoesNotExistException {
+  protected void completeFileInternal(List<Long> blockIds, long fileId, long fileLength,
```
Comment (apc999) [inline servers/src/main/java/tachyon/master/file/FileSystemMaster.java]: actually, why `protected`? I searched the code base and no one is overriding this method.

File: servers/src/main/java/tachyon/master/file/FileSystemMaster.java [hunk 3]
```diff
@@ -478,9 +474,7 @@ public long createFile(TachyonURI path, long blockSizeBytes, boolean recursive)
     }
   }
 
-  // TODO(cc) Make it protected instead of public for use in tests,
-  // after splitting up MasterInfoIntegrationTest
-  public List<Inode> createFileInternal(TachyonURI path, long blockSizeBytes, boolean recursive,
+  protected List<Inode> createFileInternal(TachyonURI path, long blockSizeBytes, boolean recursive,
```
Comment (apc999) [inline servers/src/main/java/tachyon/master/file/FileSystemMaster.java]: same here, why `protected`?

File: integration-tests/src/test/java/tachyon/master/rawtable/RawTableMasterIntegrationTest.java (hunk 1 above)
Comment (uronce-cc) [inline integration-tests/src/test/java/tachyon/master/rawtable/RawTableMasterIntegrationTest.java]: done.

File: servers/src/main/java/tachyon/master/file/FileSystemMaster.java (hunk 2 above)
Comment (uronce-cc) [inline servers/src/main/java/tachyon/master/file/FileSystemMaster.java]: I wanted to make them public only at package level, yeah, I should drop the `protected` keyword.

**Comment by uronce-cc** [comment]
@apc999 Thanks bin. I've resolved the comments.

File: integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java [hunk 4]
```diff
@@ -39,23 +39,21 @@
...
 import tachyon.thrift.FileAlreadyExistException;
 import tachyon.thrift.FileDoesNotExistException;
 import tachyon.thrift.FileInfo;
 import tachyon.thrift.InvalidPathException;
 import tachyon.thrift.SuspectedFileSizeException;
-import tachyon.thrift.TableColumnException;
 import tachyon.thrift.TachyonException;
 
 /**
  * Unit tests for tachyon.MasterInfo
```
Comment (gpang) [inline integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java]: Update this comment for this correct filename.

**Comment by gpang** [comment]
Thanks for the PR! I just had one comment.

**Comment by uronce-cc** [comment]
@gpang done!

**Comment by gpang** [comment]
LGTM

**Pull request merged**